docs/sections/user_guide/index.rst

```diff
@@ -5,6 +5,16 @@ User Guide
    :maxdepth: 2
 
    installation
+
+.. raw:: html
+
+   <ul>
+     <li><a href="https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Fexample.ipynb" target="_blank">UW Tutorial and Example Usage via Jupyter Notebook</a></li>
+   </ul>
```

Comment:
I think it would be nicer for the user if we could arrange to have uwtools installed in advance, but a less-good idea might be to have an early cell that does
```
%%bash
conda install -c ufs-community -c conda-forge --override-channels uwtools
```
which will do the necessary installation.

Reply:
It looks like after some googling that this can be a more complex way to go about this since there isn't a repo backing the notebook when Binder spins up.

Reply:
[This page](https://mybinder.readthedocs.io/en/latest/introduction.html) says
> Configuration files may be placed in the root of your repository or in a `binder/` folder in the repository’s root (i.e. `myproject/binder/`).
and then [this one](https://mybinder.readthedocs.io/en/latest/using/config_files.html#config-files) makes it sound like adding `binder/environment.yaml` (we should not put it in the repo root), perhaps just with content
```
uwtools
```
might instruct Binder to install `uwtools` in the notebook environment.

(I can't tell if this should be `binder/environment.yaml`, or `notebooks/environment.yaml`: The URL to Binder we're using is `https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Fexample.ipynb`, so I wonder if the `/notebooks` part makes that the root from Binder's perspective, in which case maybe `notebooks/environment.yaml` is the right way to go; or whether Binder will always find the repo root, in which case we'd want `binder/environment.yaml`. It would be nice if the URL made it see only the `notebooks/` directory. I mean, if this works at all.)

Reply:
Just confirming I had the same error as Paul show up when running the notebook.

Reply:
Troubleshot this issue during UW group, the solution was to add the 'binder/environment.yml' to the '/notebooks' directory so that the Binder container has the uwtools installed. This change was added to PR-533 as it needed the example notebook to test.